app/models/container_service.rb

```diff
@@ -1,5 +1,6 @@
 class ContainerService < ApplicationRecord
   include CustomAttributeMixin
+  include SupportsFeatureMixin
```

Comment:
How come I can't see the include like in the other classes? Is that transitive in some way?